packages/smart-wallet/scripts/wfup.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * @file Proposal Builder: Upgrade walletFactory
+ *
+ * Usage:
+ *   agoric run wfup.js
+ */
+
+import { makeHelpers } from '@agoric/deploy-script-support';
+import {
+  getManifestForUpgrade,
+  getManifestForGame1,
```

Comment:
Is packages/smart-wallet/scripts/ intended for production code or testing code? It appears to currently have a mixture.

Reply:
production. it's... um... intended to get the reviewer's attention :)
I'd like to talk it over.